src/Tools/Installer.php

```diff
@@ -32,11 +32,11 @@ public function install()
         $db->executeQuery('CREATE TABLE `' . Dao::TABLE_NAME . '` (
               `id` int(11) NOT NULL AUTO_INCREMENT,
               `o_id` int(11) NOT NULL,
```

Comment:
`o_id` is not replaced?

Reply:
Well this one refers to the object id on `objects` table and since this table already has an `id` column, it does not make sense to remove the `o_` prefix.